SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -161,7 +161,7 @@ mkdir -p %{buildroot}/etc/systemd/system/containerd.service.d/
 # for testing policy/snapshotter without SEV SNP we use CH (with kernel-uvm and initrd) instead of CH-CVM with IGVM
 # Note: our kata-containers config toml expects cloud-hypervisor and kernel under a certain path/name, so we align this through symlinks here
```

Comment:
Now that we're pointing both symlinks to the same CLH binary, please also update this comment or remove the parts that aren't valid anymore